### ISSUE TITLE
phantomjs failing on debian 8 machine

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/build#937]
getting this failure in multiple modules on debian 8

```
error while loading shared libraries: libfontconfig.so.1: cannot open shared object file: No such file or directory
```

**Comment by MylesBorins** [comment]
getting the same error on ubuntu 1204 as well

**Comment by refack** [comment]
This is CitGM, right?

**Comment by MylesBorins** [comment]
indeed CitGM

**Comment by gibfahn** [comment]
@MylesBorins could you give a link to a failing job?

What makes you think this is a machine problem? Which other modules are failing (just phantomjs?)

@nodejs/build has anyone changed these machines recently?

**Comment by MylesBorins** [comment]
Across multiple builds + multiple jobs + mutiple modules.

We don't test PhatomJS but rather it is a build tool in multiple stacks. All of them are failing with the same error regarding a missing lib. It is possible that this is a new dynamic dependency of the phantomjs binary

**Comment by refack** [comment]
One example (machine & package) as a thread I can start pulling?

**Comment by MylesBorins** [comment]
The phantomJS failure is due to not having [libfontconfig](https://packages.debian.org/cgi-bin/search_contents.pl?word=libfontconfig.so.1&searchmode=searchfiles&case=insensitive&version=unstable&arch=i386)

platforms exhibiting behavior

* [mocha on v6.x ubuntu1204](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/1030/nodes=ubuntu1204-64/testReport/(root)/citgm/mocha_v4_0_1/)

I also discovered that some machine are missing libunwind on some machines

[zeromq on v6.x ubuntu1604](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/1030/nodes=ubuntu1604-64/testReport/(root)/citgm/zeromq_v4_6_0/)

Here is a link to the citgm job in case you want to dig into other failures

https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/1030/

**Comment by rvagg** [comment]
This _should_ be fixed now, see #964. I couldn't find any of the libunwind failures when I went looking for logs (1030 was deleted by the time I looked). Let me know if you see that one again.

**Comment by rvagg** [comment]
re-ran citgm on v8.9.0 @ https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/1052/

all green on the two fedora boxes included (now 25 and 26 and in theory it should stay up to date with the latest two fedora releases), the failure on debian8 is unrelated (port in use)
